css-grid/Overview.bs

```diff
@@ -1147,14 +1174,15 @@ The Explicit Grid</h3>
 	together define the <dfn export local-lt="explicit">explicit grid</dfn> of a <a>grid container</a>.
 	The 'grid' property is a <a>shorthand</a> that can be used to set all three at the same time.
 	The final grid may end up larger due to <a>grid items</a> placed outside the <a>explicit grid</a>;
-	in this case, any implicit tracks are sized by the 'grid-auto-rows' and 'grid-auto-columns' properties.
+	in this case implicit tracks will be created,
+	these implicit tracks will be sized by the 'grid-auto-rows' and 'grid-auto-columns' properties.
 
 	The size of the <a>explicit grid</a> is determined by the larger of
 	the number of rows/columns defined by 'grid-template-areas'
 	and the number of rows/columns sized by 'grid-template-rows'/'grid-template-columns'.
 	Any rows/columns defined by 'grid-template-areas' but not sized by 'grid-template-rows'/'grid-template-columns'
 	take their size from the 'grid-auto-rows'/'grid-auto-columns' properties.
-	If these properties don't define <em>any</em> <a>explicit</a> tracks,
```

Comment:
I think the "explicit" here needs to be kept, to make sure there is no confusion with implicit tracks.